### DISCUSSION TITLE
Can you provide a working example of idom.web.module_from_url

### DISCUSSION THREAD

**Discussion by sylvoslee** [idom-team/idom#432] [Question] [answered]
Can you provide a working example of idom.web.module_from_url

**Comment by rmorshea** [comment, marked as answer]
An example usage can be seen [here](https://idom-docs.herokuapp.com/docs/javascript-components.html#distributing-javascript-via-cdn). However if I had to guess, that's probably not what you're looking for. If you're trying to replicate the old behavior of `idom.Module` you'd actually want to use is [`module_from_template`](https://idom-docs.herokuapp.com/docs/javascript-components.html#dynamically-loaded-components) which allows you to just pass in the name of a React-based JS package.
